python_modules/dagster/dagster_tests/definitions_tests/asset_reconciliation_tests/freshness_policy_scenarios.py

```diff
@@ -336,7 +338,22 @@
         unevaluated_runs=[run([f"asset{i}" for i in range(1, 6)])],
         evaluation_delta=datetime.timedelta(minutes=35),
         # need to run assets 1, 2 and 3 as they're all part of the same non-subsettable multi asset
-        expected_run_requests=[run_request(asset_keys=["asset1", "asset2", "asset3", "asset5"])],
+        # need to run asset 4 as it eagerly updates after asset 1
+        expected_run_requests=[
```

Comment:
the change here is in how we handle non-subsettable multi-assets (we now materialize the eager asset that is downstream of asset 1 in the same tick).

@johannkm we briefly discussed this scenario, and I still feel like either behavior is fine -- this feels slightly more consistent (asset 4 would be executed after asset 1 was fully materialized anyway, so might as well do it in the same run)